Shurgeken/Assets/Scripts/Networking/GameMaster.cs

```diff
@@ -42,12 +42,16 @@ public void blueTeamScored()
     public void redTeamScored()
     {
         redTeamScore++;
-        if (redTeamScore > scoreToWin)
+        if (redTeamScore >= scoreToWin)
         {
-            Debug.Log("Red wins. Handle it please.");
+            //Debug.Log("Red wins. Handle it please.");
+
+            GameObject.Find("UI Popup").transform.FindChild("Win").GetComponent<PopupFadeout>().StartPopup();
```

Comment:
Why not just create an update method for the GameMaster and shove it in there? It is still a monobehavior class. (Although it should be Photon.Monobehavior instead of just Monobehavior like the network controllers) Scores are synced across the network, so we can just do a win bool or something in there.

Likewise there should be logic in that update method to determine if an entire team is in the jail, although I think I need to write a little bit of code to be able to detect that.